postgresql/cluster/init.sls

```diff
@@ -0,0 +1,14 @@
+{%- from "postgresql/map.jinja" import cluster with context %}
+
+{%- if cluster.enabled %}
+include:
+{%- if cluster.get("mode") == "hot_standby" and cluster.get("role") == "master" %}
+- postgresql.cluster.hot-standby.master
+{% elif cluster.get("mode") == "hot_standby" and cluster.get("role") == "slave" %}
+- postgresql.cluster.hot-standby.slave
+{% elif cluster.get("mode") == "bdr" and cluster.get("role") == "master" %}
```

Comment:
You should include postgresql.cluster.bdr and just let it's init.sls to decide whether to include slave or master role. It's more logical using nested approach than doing this stuff in single file.